src/views/MetocDataView.vue

```diff
@@ -660,6 +670,32 @@ export default class MetocDataView extends Mixins(WMSMixin, TimeSeriesMixin) {
     }
   }
 
+  beforeRouteUpdate(to: Route, from: Route, next: NavigationGuardNext) {
```

Comment:
Het is denk ik een goed idee om bij deze functie een comment toe te voegen waarom dit gedoe nodig is.